### DISCUSSION TITLE
在Manjaro Linux 下，Markdown 查看器 无法正常查看

### DISCUSSION THREAD

**Discussion by internalerro** [tamlok/vnote#2162] [General]
## 在  Manjaro 系统下，运行 vnote 出现 markdown 可以编辑，但无法查阅的情况。
![image](https://user-images.githubusercontent.com/107781271/174464472-259484fd-523c-40d5-ad16-30b76f76eadb.png)
## 打开 debug 也是空白。
![image](https://user-images.githubusercontent.com/107781271/174464572-d77976cc-b69f-4017-aa41-787aa5f8510d.png)
# 发生问题系统及版本号：
.系统：Manjaro
.内核：5.15.48.1
.vnote版本：3.13.0

**Comment by tamlok** [comment]
启动参数加 --no-sandbox

**Reply by internalerro** [reply]
问题解决了，非常感谢
